### PR TITLE
zh-translation for about/contribute/terminology

### PR DESCRIPTION
**Pull request by helight**
zh-translation for about/contribute/terminology/index.md
ref: https://github.com/servicemesher/istio-official-translation/issues/1773

### CODE REVIEW

File: content/zh/about/contribute/terminology/index.md [hunk 1]
```diff
@@ -1,53 +1,49 @@
...
+- “Envoy sidecar” - 这样说没问题
+- “Envoy proxy” - 这样说没问题
+- “The Istio proxy” -- 最好避免这样说，除非说的是另外一个可能使用其它代理的高级场景。
+- “Sidecar”  -- 大多时候仅限于概念文档中使用
+- “Proxy” -- 只有在上下文非常容易理解的时候
 
-Related Terms:
+相关术语：
 
 - Proxy agent  - This is a minor infrastructural component and should only show
```
Comment (gorda) [inline content/zh/about/contribute/terminology/index.md]:
```suggestion
- Proxy agent  - This is a minor infrastructural component and should only show
```
忘了删掉原文

Comment (helight) [reply content/zh/about/contribute/terminology/index.md]: got

**Pull request merged**